src/features/Search/components/navigation-item/index.js

```diff
@@ -0,0 +1,16 @@
+import cx from 'classnames';
+
+import * as styles from './index.module.css';
```

Comment:
- [css modules](https://medium.com/m/global-identity?redirectUrl=https%3A%2F%2Fbetterprogramming.pub%2Fusing-css-modules-with-create-react-app-without-ejecting-6ceef85194e), create-react-app is using `*.module.css` as a pattern, but in general, once you start working with [webpack](https://webpack.js.org/) or any other module bundler, you'll see that it's you who defines the pattern. 
- With css modules, I will always recommend using class names to define the style and to not rely on the parent-children relationship since this can be fragile